source/gui/addonStoreGui/controls/messageDialogs.py

```diff
@@ -157,15 +157,15 @@ def _shouldInstallWhenAddonTooOldDialog(
 		# because the add-on is too old for the running version of NVDA.
 		"Warning: add-on is incompatible: {name} {version}. "
 		"Check for an updated version of this add-on if possible. "
-		"The last tested NVDA version for this add-on is {lastTestedNVDAVersion}, "
-		"your current NVDA version is {NVDAVersion}. "
+		"This add-on was last tested with NVDA {lastTestedNVDAVersion}. "
+		"NVDA requires this add-on to be tested with NVDA {nvdaVersion} or higher. "
 		"Installation may cause unstable behavior in NVDA.\n"
```

Comment:
```suggestion
		"Installing this add-on may cause NVDA to become unstable.\n"
```
I think this is slightly clearer.